src/lib/ViewSet.svelte

```diff
@@ -12,10 +12,10 @@
 	export let set: Set;
 	export let fontFamily: string | undefined = undefined;
 
+	const ROOTS = ['A', 'Bb', 'B', 'C', 'Db', 'D', 'Eb', 'E', 'F', 'F#', 'G', 'Ab'];
```

Comment:
Were you gonna find a better place to put this, or are you happy here? Does this just get used for displaying keys? If so can it use the proper flat/sharp symbols?